Aquarius/Src/Aquarius/Core/Input.h

```diff
@@ -0,0 +1,168 @@
+#include <cstdint>
+
+#include "Window.h"
+
+namespace Aquarius {
+	enum class KeyState : int
+	{
+		Released = 0,
+		Pressed = 1
+	};
+
+	using MouseButtonState = KeyState;
+
+	enum class KeyCode : int
+	{
```

Comment:
Make a mention that these are all from glfw3.h

Reply:
Yup good call